src/main/java/de/qabel/desktop/hockeyapp/HockeyCrashesClient.java

```diff
@@ -0,0 +1,56 @@
+package de.qabel.desktop.hockeyapp;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.apache.http.entity.mime.content.ByteArrayBody;
+
+import java.io.IOException;
+import java.util.Date;
+
+public class HockeyCrashesClient implements CrashesClient {
+
+    private HockeyAppRequestBuilder requestBuilder;
+    private VersionClient versionClient;
+
+    public HockeyCrashesClient(HockeyAppRequestBuilder requestBuilder, VersionClient versionClient) {
+        this.requestBuilder = requestBuilder;
+        this.versionClient = versionClient;
+    }
+
+    @Override
+    public void sendStacktrace(String feedback, String stacktrace) throws IOException {
+        HttpPost request = requestBuilder.getHttpPost("/crashes/upload");
+
+        String operatingSystemInformation = System.getProperty("os.name") + " / " + System.getProperty("os.arch") + " / " + System.getProperty("os.version");
+        String manufacturer = System.getProperty("java.vendor");
+        String model = System.getProperty("java.version");
```

Comment:
If this were Kotlin code, I'd say this should be extracted into a data class…

Here in plain old Java this is probably too much hassle, so this is fine.